src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -66,6 +67,7 @@ def load_arguments(self, _):
         c.argument('disable_proxy', options_list=['--disable-proxy'], arg_group='Proxy', action='store_true', help='Disables proxy settings for agents')
         c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
         c.argument('container_log_path', help='Override the default container log path to enable fluent-bit logging')
+        c.argument('enable_oidc_issuer', options_list=['--enable-oidc-issuer'], help="Enable creation of OIDC issuer endpoint used for workload identity")
```

Comment:
aren't we supposed to keep this in enable/disable features cmdlet? how will the customer disable this feature?

Reply:
by explicitly giving --enable-oidc-issuer false in the update